1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -5,19 +5,28 @@
 */
 
 const BIRTHDAYS = [
-    "January 7th",
-    "February 12th",
-    "April 3rd",
-    "April 5th",
-    "May 3rd",
-    "July 11th",
-    "July 17th",
-    "September 28th",
-    "November 15th"
+  "January 7th",
+  "February 12th",
+  "April 3rd",
+  "April 5th",
+  "May 3rd",
+  "July 11th",
+  "July 17th",
+  "September 28th",
+  "November 15th",
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  // TODO
+  let i = 0;
+  let birthday = "";
+  while (i < BIRTHDAYS.length) {
+    if (BIRTHDAYS[i].includes("July")) {
+      birthday = BIRTHDAYS[i];
+      return birthday;
+    }
+    i++;
```

Comment:
Yes, you have the working function. We could simplify the function by removing the use of **birthday** variable at line 22 and return the answer directly.